### ISSUE TITLE
Repository not found

### ISSUE THREAD

**Issue by fmnhExhibits** [zooniverse/Leaves#1]
I'm getting this error when running npm install

```
npm ERR! Command failed: git clone --template=/Users/fmnh/.npm/_git-remotes/_templates --mirror git://github.com/chrissnyder/stack-of-pages.git /Users/fmnh/.npm/_git-remotes/git-github-com-chrissnyder-stack-of-pages-git-2c271ace
npm ERR! Cloning into bare repository '/Users/fmnh/.npm/_git-remotes/git-github-com-chrissnyder-stack-of-pages-git-2c271ace'...
npm ERR! fatal: remote error: 
npm ERR!   Repository not found.

```

Is there any way to get access to the stack-of-pages repo? Or another way around this issue

**Comment by srallen** [comment]
I updated the `package.json` to use the `stack-of-pages` that is published on npmjs.com instead of the github repo it was pointed to. It seems that github repo was deleted or made private. Either way, the project seems to work with version 0.1.5. I'm don't know why the project was using that forked version instead of the published npm version since I wasn't a part of the zooniverse at that time.

@brian-c I also noticed that `stack-of-pages` doesn't have the zooniverse owner on npm. I think this is one that got missed in the transfer to zooniverse ownership.
